installation/resources/installer-config-gke-benchmark.yaml.tpl

```diff
@@ -31,8 +31,18 @@ data:
   global.cockpit.auth.idpHost: "https://container.googleapis.com/v1/projects/$CLOUDSDK_CORE_PROJECT/locations/$CLOUDSDK_COMPUTE_ZONE/clusters/$COMMON_NAME"
-  global.cockpit.auth.path: "/jwks"
\ No newline at end of file
+  global.cockpit.auth.path: "/jwks"
+  global.tenantFetcher.endpoints.subaccountCreated: "http://compass-external-services-mock.compass-system.svc.cluster.local:8080/tenant-fetcher/subaccount-create"
+  global.tenantFetcher.secret.name: "compass-tenant-fetcher-secret"
+  global.tenantFetcher.secret.clientIdKey: "client-id"
+  global.tenantFetcher.secret.oauthMode: "oauth-mtls"
+  global.tenantFetcher.secret.clientCertKey: "client-cert"
+  global.tenantFetcher.secret.clientKeyKey: "client-key"
+  global.tenantFetcher.secret.oauthUrlKey: "url"
+  global.tenantFetcher.secret.skipSSLValidation: "true"
```

Comment:
We have the **exact same** values in installer-config-local.yaml.tpl and in Value.yaml. Why do we replicate them? It should be enough for these values to be maintained only in Values.yaml. Also in Values.yaml the values are not quoted which may be a problem.

Reply:
<img width="1405" alt="Screen Shot 2022-04-21 at 14 27 34" src="https://user-images.githubusercontent.com/39959090/164450957-5d81e11d-fb2f-4dfe-8029-700446250ba9.png">